wolfssl/ssl.h

```diff
@@ -3013,6 +3013,8 @@ WOLFSSL_API int                  wolfSSL_GetCipherBlockSize(WOLFSSL* ssl);
 WOLFSSL_API int                  wolfSSL_GetAeadMacSize(WOLFSSL* ssl);
 WOLFSSL_API int                  wolfSSL_GetHmacSize(WOLFSSL* ssl);
 WOLFSSL_API int                  wolfSSL_GetHmacType(WOLFSSL* ssl);
+WOLFSSL_API int                  wolfSSL_GetPeerSequenceNumber(WOLFSSL*, word64*);
```

Comment:
Make sure and wrap these with the same macro. Also we are using the argument names in the API def now, so please add those `WOLFSSL* ssl, word64 *seq`. Thanks

Reply:
Added `#ifdef WORD64_AVAILABLE` and the argument names